dev/tests/js/jasmine/tests/app/code/Magento/AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/overlay.test.js

```diff
@@ -0,0 +1,283 @@
+
+        var overlayObj;
+
+        beforeEach(function () {
+            overlayObj = new Overlay({
+                provider: 'providerName',
+                licensed: {}
+            });
+        });
+
+        describe('"initObservable" method', function () {
+
+            it('Check for defined ', function () {
+                expect(overlayObj.hasOwnProperty('initObservable')).toBeDefined();
+            });
+
+            it('Check method type', function () {
+                var type = typeof overlayObj.initObservable;
+
+                expect(type).toEqual('function');
+            });
+
+            it('licensed will be defined', function () {
+                expect(overlayObj.licensed).toBeDefined();
+            });
+        });
+
+        describe('"updateLicensed" method', function () {
+
+            var handler = {
+                success: function(data) {
+                    return [1,2]
+                },
+                fail: function(data) {
+                    return 'Error Message';
+                }
+            };
+            var testObj = {
+                ajaxFunction: function() {
+                    var promise =
+                        $.ajax({
+                            url: 'http://example.com/admin/adobe_stock/license/getlist/?ids=1,2,3',
+                            type: 'GET',
+                            data: {
+                                'form_key': 'MagentoFormkey'
+                            },
+                            dataType: 'json',
+                            context: overlayObj
+                        });
+                    promise.done(function(data, status) {
+                        handler.success(data);
+                    });
+                    promise.fail(function(status, error) {
+                        handler.fail(error);
+                    });
+                }
+            };
+
+            it('Check for defined', function () {
+                expect(overlayObj.hasOwnProperty('updateLicensed')).toBeDefined();
+            });
+
+            it('Check method type', function () {
+                var type = typeof overlayObj.updateLicensed;
+               expect(type).toEqual('function');
+            });
+
+            it("Check Ajax Success request", function() {
+
+                spyOn(handler, 'success').and.callFake(function(e) {
+                    return [1,2]
+                });
+                spyOn($, 'ajax').and.callFake(function(e) {
+                    return $.Deferred().resolve({
+                        'response': 'success'
+                    }).promise();
+                });
+                testObj.ajaxFunction();
+                expect(handler.success).toHaveBeenCalled();
+                expect(typeof handler.success()).toBe('object');
+                expect(handler.success().length).toBe(2);
+            });
+
+            it('Check Ajax failure request', function() {
```

Comment:
Same comments as for the ajax success test above, this one should invoke `Overlay.updateLicensed`, and `$.ajax` should be spied on to ensure it returns proper "ajax failure" data.